File: app/src/main/java/com/angogasapps/myfamily/ui/screens/news_center/NewsCenterActivity.java
package com.angogasapps.myfamily.ui.screens.news_center;

import androidx.appcompat.app.AppCompatActivity;
import com.angogasapps.myfamily.databinding.ActivityNewsCenterBinding;

import android.content.Intent;
import android.os.Bundle;

import com.angogasapps.myfamily.R;

public class NewsCenterActivity extends AppCompatActivity {
    ActivityNewsCenterBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityNewsCenterBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        binding.createNewNewsBtn.setOnClickListener(v -> {
            startActivity(new Intent(NewsCenterActivity.this, CreateNewNewsActivity.class));
        });

        startActivity(new Intent(this, CreateNewNewsActivity.class));
        finish();
    }
}

File: app/src/main/java/com/angogasapps/myfamily/ui/screens/news_center/NewsCenterActivity.kt
package com.angogasapps.myfamily.ui.screens.news_center

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.content.Intent
import android.view.View
import com.angogasapps.myfamily.databinding.ActivityNewsCenterBinding
import com.angogasapps.myfamily.ui.screens.news_center.CreateNewNewsActivity

class NewsCenterActivity : AppCompatActivity() {
    lateinit var binding: ActivityNewsCenterBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityNewsCenterBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.createNewNewsBtn.setOnClickListener {
            startActivity(
                Intent(
                    this@NewsCenterActivity,
                    CreateNewNewsActivity::class.java
                )
            )
        }
        startActivity(Intent(this, CreateNewNewsActivity::class.java))
        finish()
    }
}